### PR TITLE
[Opt](ShortCircuit) add more stats info to trace

### PR DESCRIPTION
**Pull request by eldenmoon**

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -886,13 +886,10 @@ void PInternalService::fetch_arrow_flight_schema(google::protobuf::RpcController
...
-    RETURN_IF_ERROR(lookup_util.init(request, response));
-    RETURN_IF_ERROR(lookup_util.lookup_up());
-    if (VLOG_DEBUG_IS_ON) {
-        VLOG_DEBUG << lookup_util.print_profile();
-    }
-    LOG_EVERY_N(INFO, 500) << lookup_util.print_profile();
+    PointQueryExecutor executor;
+    RETURN_IF_ERROR(executor.init(request, response));
+    RETURN_IF_ERROR(executor.lookup_up());
+    executor.print_profile();
```
Comment (xiaokang) [inline be/src/service/internal_service.cpp:892]: print profile for every request?

File: be/src/service/point_query_executor.cpp [hunk 2]
```diff
@@ -380,6 +396,17 @@ Status PointQueryExecutor::_lookup_row_key() {
         specified_rowsets = _tablet->get_rowset_by_ids(nullptr);
     }
     std::vector<std::unique_ptr<SegmentCacheHandle>> segment_caches(specified_rowsets.size());
+    // init segment_cache
```
Comment (xiaokang) [inline be/src/service/point_query_executor.cpp:399]: Why add this logic?

File: be/src/service/internal_service.cpp (hunk 1 above)
Comment (eldenmoon) [reply be/src/service/internal_service.cpp:892]: print profile will decide whether to print

File: be/src/service/point_query_executor.cpp (hunk 2 above)
Comment (eldenmoon) [reply be/src/service/point_query_executor.cpp:399]: in order to trace the time of loading segments

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**